src/gui/tray/TalkReplyTextField.qml

```diff
@@ -37,6 +28,8 @@ Item {
         width: 250
 
         onAccepted: root.sendReplyMessage()
+        onTextEdited: sendReplyMessageButton.enabled = true
```

Comment:
@camilasan What if the text will get deleted after being edited? `sendReplyMessageButton.enabled` will not get reset, right? This sounds a bit complicated. Why not add binding of `Text.text !== ""` to `sendReplyMessageButton.enabled`?